packages/ui/src/components/organisms/RatingSummary/styles.scss

```
@@ -0,0 +1,121 @@
+[data-fs-rating-summary] {
+  // --------------------------------------------------------
+  // Design Tokens for Rating Summary
+  // --------------------------------------------------------
+
+  --fs-rating-summary-width                               : 30%;
+  --fs-rating-summary-min-height                          : 15rem;
+  --fs-rating-summary-line-height                         : 1;
+  --fs-rating-summary-gap                                 : var(--fs-spacing-4);
+  --fs-rating-summary-padding                             : var(--fs-spacing-5);
+  --fs-rating-summary-font-size                           : var(--fs-text-size-0);
+  --fs-rating-summary-font-weight                         : var(--fs-text-weight-regular);
+  --fs-rating-summary-border-radius                       : var(--fs-border-radius);
+  --fs-rating-summary-background-color                    : var(--fs-color-neutral-1);
+
+  --fs-rating-header-vertical-gap                         : var(--fs-spacing-0);
```

```suggestion
  --fs-rating-summary-header-vertical-gap                         : var(--fs-spacing-0);
```

suggestion: rename this tokens to add `summary`